spec/mailers/registration_mailer_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe RegistrationMailer, type: :mailer do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
this was automatically created. Not sure whether to delete or figure out how to write tests.